interop-testing/src/main/java/io/grpc/testing/integration/XdsTestServer.java

```diff
@@ -162,11 +174,15 @@ private void parseArgs(String[] args) {
               + s.secureMode
               + "\n  --server_id=STRING  server ID for response."
               + "\n                      Default: "
-              + s.serverId);
+              + s.serverId
+              + "\n  --address_type=STRING  type of IP address to bind to (IPV4|IPV6|IPV4_IPV6)."
```

Comment:
Will there be an update to https://github.com/grpc/grpc/blob/master/doc/xds-test-descriptions.md ?